.teamcity/performance-test-runtimes.json

```diff
@@ -253,6 +253,12 @@
     "testProject" : "santaTrackerAndroidBuild",
     "linux" : 413000
   } ]
+}, {
+  "scenario" : "org.gradle.performance.regression.android.FasterIncrementalAndroidBuildsPerformanceTest.file system watching baseline non-abi change (build comparison)",
+  "runtimes" : [ {
```

Comment:
I just realized that these should probably be called `durations` not `runtimes`, no? Or perhaps just `duration`. Or `executionTimes`. The word `runtimes` especially spelled the way it is brings to mind runtime dependencies and Java runtimes.

Reply:
I renamed runtimes to duration in the file and in the code.